pages/index.js

```diff
@@ -31,7 +31,7 @@ const StyledList = styled.ul`
 `;
 
 const StyledFixLink = styled(Link)`
-  position: fixed;
-  right: 1.125rem;
-  bottom: 1.125rem;
+  position: sticky;
+  float: right;
```

Comment:
'float' is a layout construct from the early days of CSS. I would not recommend to use it anywhere. It has has a lot of missbehaviours in modern websites. You don't need to use it. How can position a fixed (or even a sticky) element inside of its parent???

Reply:
...combination of reletaive (main) and absolute, but we want , that the add-button scrolls with the page

Reply:
Your intention with wrapping the Link in a container was right. Here's the complete solution:
```js
      <StyledLinkContainer>
        <StyledFixLink href="/createActivity">
          <Icon name="add" />
        </StyledFixLink>
      </StyledLinkContainer>
    </>
  );
}
const StyledLinkContainer = styled.div`
  display: flex;
  justify-content: flex-end;
`;

const StyledFixLink = styled(Link)`
  position: fixed;
  bottom: 30px;
`;
```